src/server.cc

```diff
@@ -605,6 +622,12 @@ void Server::GetRocksDBInfo(std::string *info) {
   string_stream << "num_background_errors:" << num_backgroud_errors << "\r\n";
   string_stream << "flush_count:" << storage_->GetFlushCount()<< "\r\n";
   string_stream << "compaction_count:" << storage_->GetCompactionCount()<< "\r\n";
+  string_stream << "written_per_sec:" << stats_.GetInstantaneousMetric(STATS_METRIC_ROCKSDB_PUT) << "\r\n";
```

Comment:
why did you use **written** word?

how about
`put_per_sec` `get_per_sec`
or 
`put_ops_per_sec` `get_ops_per_sec`
especially, for `next_per_sec` and `pre_per_sec`, i prefer to use `next_ops_per_sec`, `prev_ops_per_sec`, just using `next` or `pre` is not clear i think.

Reply:
`ops` means `operation per second`, same thing as `per_sec`.
the words of `put` or `get` is great

Reply:
`ops` just is the short of operations
the reason why I want to add `ops` is that `pre` or `next` is not clear, the `ops` can know they are operations.
Currently, both are ok for me, @git-hulk WDYT?

Reply:
Is the abbr `ops` confusing? The `ops` in `instantaneous_ops_per_sec` confused me at start.

Reply:
I think `put_per_sec` should be better than `get_ops_per_sec` since the `ops` may be confused with the instantaneous_ops_per_sec.